### PR TITLE
Correct default values in stratified_thermal_storage.py

### PR DESCRIPTION
**Pull request by MaGering**
With this pull request the default values for the density and capacity are corrected in the functions 

- calculate_capacities() and 
- calculate_losses() 

in stratified_thermal_storage.py. As a simplification the values are calculated with CoolProp for a constant temperature of 80 °C.

### CODE REVIEW

**Comment by MaGering** [comment]
This approach has already been discussed in the following conversation:

https://github.com/oemof/oemof-thermal/pull/19#discussion_r352484326

File: src/oemof/thermal/stratified_thermal_storage.py [hunk 1]
```diff
@@ -112,9 +112,11 @@ def calculate_capacities(
 
     heat_capacity: numeric
         Average specific heat capacity of storage medium [J/(kg*K)]
+        Calculated with CoolProp at a constant temperature of 80 °C as a simplification
```
Comment (jnnr) [inline src/oemof/thermal/stratified_thermal_storage.py]:
```suggestion
        Default values calculated with CoolProp for a temperature of 80 °C as a simplifying assumption
```

**Review by jnnr — COMMENTED** [review]
Looks good, thanks! I suggest to change the text slightly, referring to the default values.

File: src/oemof/thermal/stratified_thermal_storage.py (hunk 1 above)
Comment (MaGering) [reply src/oemof/thermal/stratified_thermal_storage.py]: Thank you for your suggestion! I've adjusted the text accordingly.

**Review by jnnr — APPROVED** [review]

**Pull request merged**